internal/testutility/utility.go

```diff
@@ -34,11 +34,17 @@ func Skip(t *testing.T, args ...any) {
 	snaps.Skip(t, args...)
 }
 
+// Access to environment variable that toggles acceptance testing execution paths
+// Acceptance testing is "On" only when var set to "true"
+func AcceptanceTestOn() bool {
```

Comment:
nit: I feel like it might be nicer to name this something like `IsAcceptanceTesting`

This could just be me though so I'm more mentioning this to see if anyone else feels similar but if not it can stay - I think I'm just not use to "On" being used as the main expression of a function being a conditional 🤷

Reply:
I have no strong feelings about this and couldnt see a pattern in the existing things, figured your comment if you cared and had a better suggestionm is is fine for me, will do.